### PR TITLE
Release Notes v1.49

### PR DESCRIPTION
**Pull request by jshaughn**

### CODE REVIEW

File: content/en/news/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,27 @@ weight: 1
...
+Sprint Release: April 8th, 2022
+
+Features:
+
+* [Auth: Phase out usage of JWTs](https://github.com/kiali/kiali/issues/4542)
+* [Kiali UI and Kiali Server can point to a single commit](https://github.com/kiali/kiali/issues/4895)
+* [Add cypress UI tests around the Overview page](https://github.com/kiali/kiali/issues/4872)
+* [Update Prometheus client lib](https://github.com/kiali/kiali/issues/4884)
+* [Transfer frontend repo into kiali repo](https://github.com/kiali/kiali/issues/4825)
+* [Hack script to create a Kind cluster in CI](https://github.com/kiali/kiali/issues/4833)
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md:19]: do we care to call out hack scripts?

File: content/en/news/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,27 @@ weight: 1
...
+## 1.49.0
+Sprint Release: April 8th, 2022
+
+Features:
+
+* [Auth: Phase out usage of JWTs](https://github.com/kiali/kiali/issues/4542)
+* [Kiali UI and Kiali Server can point to a single commit](https://github.com/kiali/kiali/issues/4895)
+* [Add cypress UI tests around the Overview page](https://github.com/kiali/kiali/issues/4872)
+* [Update Prometheus client lib](https://github.com/kiali/kiali/issues/4884)
+* [Transfer frontend repo into kiali repo](https://github.com/kiali/kiali/issues/4825)
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md:18]: is this really a feature?  Do we want this in the release notes?

File: content/en/news/release-notes.md [hunk 3]
```diff
@@ -6,6 +6,27 @@ weight: 1
...
+* [Hack script to create a Kind cluster in CI](https://github.com/kiali/kiali/issues/4833)
+* [Feature flag to disable log browser](https://github.com/kiali/kiali/issues/4737)
+* [Support Gateways workloads in user namespaces](https://github.com/kiali/kiali/issues/3408)
+* [add the ability to add annotations to configmap.yaml](https://github.com/kiali/kiali/issues/4814)
+
+Fixes:
+
+* [ui crash with no gateways](https://github.com/kiali/kiali/issues/4892)
+* [UI messages at INFO level look just like ERROR level messages](https://github.com/kiali/kiali/issues/4871)
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md:27]: while this really was a bug, it turns out nowhere in the UI (today) uses INFO level popup messages (toast msg I think they are called). Do we care to call this out here?

**Review by jmazzitelli — APPROVED** [review]
I have a few comments, but nothing that would prevent merge. I think some of these can be removed, though, hence my comments

File: content/en/news/release-notes.md (hunk 1 above)
Comment (jshaughn) [reply content/en/news/release-notes.md:19]: I dunno, could be useful to a community dev.  doesn't hurt.  let's not overthink this :)

File: content/en/news/release-notes.md (hunk 2 above)
Comment (jshaughn) [reply content/en/news/release-notes.md:18]: I dunno, could be useful to a community dev.  doesn't hurt.  let's not overthink this :)

File: content/en/news/release-notes.md (hunk 3 above)
Comment (jshaughn) [reply content/en/news/release-notes.md:27]: I dunno,  doesn't hurt.  let's not overthink this :)

**Pull request merged**